sway-book/src/documentation/misc/workarounds/missing-features.md

```diff
@@ -0,0 +1,14 @@
+# Missing Features
+
+> TODO: also copied from the current book (some old version). Needs work
+
+- [Issue: #1182](https://github.com/FuelLabs/sway/issues/1182)
+  - Arrays in a storage block are not yet supported. See the [Manual Storage Management](https://fuellabs.github.io/sway/v0.24.5/blockchain-development/storage.html#manual-storage-management) section for details on how to use `store` and `get` from the standard library to manage storage slots directly. Note, however, that `StorageMap<K, V>` does support arbitrary types for `K` and `V` without any limitations.
+- [Issue: #428](https://github.com/FuelLabs/sway/issues/428)
+  - Arrays are currently immutable which means that changing elements of an array once initialized is not yet possible.
+- [Issue: #1796](https://github.com/FuelLabs/sway/issues/1796)
```

Comment:
I don't think this is the correct issue.  I think we want this [one](https://github.com/FuelLabs/sway/issues/2465)